app/_src/gateway/production/monitoring/readiness-check.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Readiness Check
+content_type: tutorial
+---
+
+This tutorial will guide you through the process of using the Node Readiness Endpoint, which provides a reliable and simple way to determine if Kong is ready to serve user requests.
+
+The readiness check endpoint returns a `200 OK` response when Kong is ready, or a `503 Service Temporarily Unavailable` response when it's not. This is useful for load balancers and other tools that need to monitor the readiness of Kong instances.
+
+Note that the readiness endpoint does not return detailed node readiness information such as the
+amount of memory Kong node is using. You might want to check out the [`/status` API](/gateway/{{page.kong_version}}/admin-api/#retrieve-node-status) as well.
+
+## Prerequisites
+
+* Kong installed
+* Basic understanding of Kong configuration and deployment modes (Traditional, DB-less, and Hybrid)
+
+## Understanding the Node Readiness Endpoint
+
+Before diving into the steps, it's important to understand the purpose of the Node Readiness Endpoint and how it determines whether a Kong instance is ready or not.
+
+### Traditional mode
+
+In Traditional mode, the endpoint returns `200 OK` when all of the following conditions are met:
+
+1. Successful connection to the database
+2. Successful initial build of all routers for all workers
+3. Successful initial build of all plugin iterators for all workers
+
+### Hybrid mode (`data_plane` role) or DB-less mode
```

Comment:
What does this endpoint do for `control_plane`?

Reply:
I added another section for this (d50ba7a858). It's a little awkward with only 1 list item :shrug: but I just wanted to keep it consistent with the other sections.